flexbox/src/main/java/com/google/android/flexbox/FlexboxLayout.java

```diff
@@ -223,8 +223,8 @@ public FlexboxLayout(Context context, AttributeSet attrs, int defStyleAttr) {
         mFlexWrap = a.getInt(R.styleable.FlexboxLayout_flexWrap, FlexWrap.NOWRAP);
         mJustifyContent = a
                 .getInt(R.styleable.FlexboxLayout_justifyContent, JustifyContent.FLEX_START);
-        mAlignItems = a.getInt(R.styleable.FlexboxLayout_alignItems, AlignItems.STRETCH);
-        mAlignContent = a.getInt(R.styleable.FlexboxLayout_alignContent, AlignContent.STRETCH);
+        mAlignItems = a.getInt(R.styleable.FlexboxLayout_alignItems, AlignItems.FLEX_START);
```

Comment:
Can this break existing apps? Should we bump up the major version if so?

Reply:
Yes, it may break the existing apps and we should bump the major version once we release it.
I think it's better in the long run because setting the AlignItems.STRETCH is expensive like setting the weight attribute in LinearLayout especially FlexboxLayout is nested than several levels.

Reply:
I see. It makes sense.